braid/base.c

```diff
@@ -92,7 +92,7 @@ _braid_BaseStep(braid_Core       core,
    {
       _braid_CoreElt(core, tnext) = _braid_CoreElt(core, tstop);
    }
-   timer = _braid_MPI_Wtime(core);
+   timer = _braid_MPI_Wtime(core, 2);
```

Comment:
What's going on with this timing level thing?

Reply:
Hi Rob, the timing level thing is something that I implemented.  Basically, the user can set two timing levels
1:  old behavior of Braid, where each iteration is timed and the whole solve in braid_Drive is timed.
2:  newer, more intrusive timing of user functions (primarily for TorchBraid).  When this is turned on, output files are created with detailed timing info for each MPI rank.

The "2" here passed to _braid_MPI_Wtime(...) tells braid to only call MPI_Wtime(..) if the timing level is at 2 or greater.   This way, we only call MPI_Wtime(...) when the user has asked for this more detailed information.  

When we did the timer PR a while back, I think we decided to wrap MPI_Wtime(), so that we only called it when needed.  This seemed reasonable -- but can always be improved :-)

Reply:
Okay.  After I asked the question, I had a feeling this was not something David added, but that he was probably behind `master`?  If so, he should do a `git merge --no-ff master` into his branch before we finish the review and merge the PR.  I am happy with the timing level stuff.  It just didn't make sense to me in the context of this PR.  Thanks!

Reply:
Cool!  Sounds good.